index.js

```diff
@@ -220,12 +248,50 @@ function mine() {
 
 function fluctuate() {
     const priceSave = price;
+    var goldTotal = 0;
+    var meatTotal = 0;
+    //var currentTime = new Date().getTime() ;
+    //var timeSinceOrigin = (parseFloat(originTime) - currentTime)/-1000;
+    var userData;
+    if (Object.keys(ledger).length > 0)
+        Object.keys(ledger).forEach(function(key) {
+            userData = ledger[key];
+            goldTotal += parseFloat(userData.gold.toFixed(2));
+            meatTotal += parseFloat(userData.meatCoin.toFixed(2));
+          });
+    //Adjust by random fluctuations, trending down for deflation:
     const priceAdjust = 1 + Math.random()/75.0;
-    if (Math.random() > 0.5)
+    var adjuster = Math.random();
+    var hyperAdjuster;
+    if (adjuster < 0.6)
         price /= priceAdjust;
-    else
+    //Rare chance of large "hyper" fluctuation
+    else if (adjuster > .99) {
```

Comment:
I thought so at first, but its not actually that large of a change. At the very worst it double value of the market, but more likely increases or decreases by .25 . Also, not every 100 sec since it only fluctuates every 5 seconds. So about every 500 seconds, or about 8 min. Still quite often though. Ill change to maybe 16 min. Chaotic but good.